src/NUnitFramework/framework/Attributes/ValuesAttribute.cs

```diff
@@ -96,7 +96,7 @@ public ValuesAttribute(params object[] args)
         /// <summary>
         /// Get the collection of _values to be used as arguments
         /// </summary>
-        public IEnumerable GetData(ParameterInfo parameter)
+        public IEnumerable GetData(Type fixtureType, ParameterInfo parameter)
```

Comment:
I believe that there are two reasons:
* Generating name of the test
* Reflecting a datasource for unit test
  * I saw a data driven test defined in base class which just implemented data sources on inherited class which were not even defined as virtual on base class - so it was rather kind of convention